src/Core/Application/Security/UseCase/UpdateSecurityPolicy/UpdateSecurityPolicy.php

```diff
@@ -45,7 +50,16 @@ public function __invoke(
         UpdateSecurityPolicyPresenterInterface $presenter,
         UpdateSecurityPolicyRequest $request
     ): void {
-        $this->repository->updateSecurityPolicy(SecurityPolicyFactory::createFromRequest($request));
+        $this->debug('Updating Security Policy');
```

Comment:
according to a discussion with @callapa , information about progress should be `info` instead of `debug`